api/net/Areas/Services/Controllers/AVOverviewController.cs

```diff
@@ -18,10 +18,10 @@ namespace TNO.API.Areas.Services.Controllers;
 [ApiController]
 [Area("services")]
 [ApiVersion("1.0")]
-[Route("api/v{version:apiVersion}/[area]/reports/av/overviews")]
```

Comment:
Don't bother renaming the API endpoint path.

Reply:
Updated, thanks